scanpipe/models.py

```diff
@@ -1567,7 +1575,7 @@ def children(self, codebase=None):
         https://github.com/nexB/commoncode/blob/main/src/commoncode/resource.py
 
         `codebase` is not used in this context but required for compatibility
-        with the commoncode.resource.VirtualCodebase class API.
+        with the commoncode.resource.Codebase class API.
         """
         exactly_one_sub_directory = "[^/]+$"
```

Comment:
@tdruez There is an issue when we cannot visit the children of the "." CodebaseResource since the logic of `CodebaseResource.walk()` depends on the parent and child CodebaseResources having the same path prefixes. I'm not sure on how to write the regex to walk from "." to a subdirectory or file.

Reply:
I've fixed this by using the regex "^[^/]+$" to capture anything without a "/" in the path, e.g.

```
foobar.txt
foobar
```